pulp_smash/utils.py

```diff
@@ -67,8 +67,29 @@ def reset_pulp(server_config):
     prefix = '' if client.run(('id', '-u')).stdout.strip() == '0' else 'sudo '
     client.run('mongo pulp_database --eval db.dropDatabase()'.split())
     client.run('sudo -u apache pulp-manage-db'.split())
-    client.run((prefix + 'rm -rf /var/lib/pulp/content/*').split())
-    client.run((prefix + 'rm -rf /var/lib/pulp/published/*').split())
+    client.run((prefix + 'rm -rf /var/lib/pulp/content').split())
+    client.run((prefix + 'rm -rf /var/lib/pulp/published').split())
 
     for service in services:
         service.start()
+
+
+def reset_squid(server_config):
```

Comment:
Consider adding this function in a separate commit. It is easier to revert commits if each contains one logical change, among other benefits.